src/html/jsdoc.rs

```diff
@@ -656,5 +784,26 @@ mod test {
       parse_links("foo {@linkcode unknownSymbol} bar", &render_ctx),
       "foo `unknownSymbol` bar"
     );
+
+    #[cfg(not(target_os = "windows"))]
+    {
+      assert_eq!(
+        parse_links("foo {@link bar} bar", &render_ctx),
+        "foo [bar](../../.././/a.ts/~/bar.html) bar"
+      );
+      assert_eq!(
+        parse_links("foo {@linkcode bar} bar", &render_ctx),
+        "foo [`bar`](../../.././/a.ts/~/bar.html) bar"
+      );
+
+      assert_eq!(
+        parse_links("foo {@link module:b.ts.baz} bar", &render_ctx),
```

Comment:
This is concerning. It is not immediately clear whether this means `baz` export from `b.ts`, or `baz.ts` export from `b`, or the entire `b.ts.baz` module. Ie you can not construct the link without knowing the context of which files are around you.